azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/storage/StorageProperties.java

```diff
@@ -32,4 +32,8 @@
     @Getter
     @Setter
     private boolean allowTelemetry = true;
+    
+    @Getter
+    @Setter
+    private boolean enableHttps = true;
```

Comment:
This will change the default behavior compared with previous behavior, i.e., previously, `http` is used, but after this change, `https` will be used. How about set this `enableHttps=false` by default? To avoid affecting other users.